images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -111,7 +97,25 @@ Push-Location -Path $sdk.FullName
     "add-ons;addon-google_apis-google-21" `
     "cmake;3.6.4111459" `
     "cmake;3.10.2.4988404" `
-    "patcher;v4"
+    "patcher;v4" `
+    "ndk;$ndk_version"
```

Comment:
I don't think that we should install ndk this way. I believe it is better to just use `ndk-bundle`: https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/installers/1804/android.sh#L45

It will picked up the latest version and we won't need to hardcode it

Reply:
Agree, I have tested that ndk-bundle contains only lastest version of NDK. I have changed it.